### ISSUE TITLE
🛠️ Repo: Don't create release PR until there's a user-facing change

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [mochajs/mocha#5190]
### Tooling Suggestion Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have read and agree to Mocha's [Code of Conduct](https://github.com/mochajs/mocha/blob/main/.github/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/mochajs/mocha/blob/main/.github/CONTRIBUTING.md)
- [X] I have searched for [related issues](https://github.com/mochajs/mocha/issues?q=is%3Aissue) and [issues with the `faq` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Afaq%20), but none matched my issue.
- [X] I want to provide a PR to resolve this

### Overview

Following #5186: we're seeing a release-it PR created when the only commits on `main` are from `chore: ...` commits (`### 🧹 Chore`). #5189 is an example.

I'd think we'd want to skip creating the PR until there's something like a `feat` or `fix` that impacts users. Is such a thing doable with release-it?

### Additional Info

A note from @voxpelli's https://github.com/mochajs/mocha/pull/5189#issuecomment-2271538878:

> @JoshuaKGoldberg We can remove parts by changing / adding / removing to
> 
> https://github.com/mochajs/mocha/blob/5c2989fcc7ae17618d9db16d7c99e23dfb1d38ee/.github/release-please/config.json#L5-L17

**Comment by voxpelli** [comment]
If we change to `"hidden": false` we can hide it from the release notes

The default for `release-please` is actually:

```js
    { type: 'feat', section: 'Features' },
    { type: 'feature', section: 'Features' },
    { type: 'fix', section: 'Bug Fixes' },
    { type: 'perf', section: 'Performance Improvements' },
    { type: 'revert', section: 'Reverts' },
    { type: 'docs', section: 'Documentation', hidden: true },
    { type: 'style', section: 'Styles', hidden: true },
    { type: 'chore', section: 'Miscellaneous Chores', hidden: true },
    { type: 'refactor', section: 'Code Refactoring', hidden: true },
    { type: 'test', section: 'Tests', hidden: true },
    { type: 'build', section: 'Build System', hidden: true },
    { type: 'ci', section: 'Continuous Integration', hidden: true }
```

**Comment by JoshuaKGoldberg** [comment]
> hide it from the release notes

Ah, we've included chores in the release notes for previous ones...

**Comment by voxpelli** [comment]
Yeah, I would have wanted a way to say "Don't trigger, but include when triggered by something else"

I would ultimately want to swap to https://github.com/release-it/release-it/issues/1126, whenever I get time to implement that :/

**Comment by mark-wiemer** [comment]
Low ROI, esp high maintenance cost, closing as won't fix
